### PR TITLE
Add list barcodes page

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListTagBarcodesController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import uk.ac.bbsrc.tgac.miso.core.data.TagBarcodeFamily;
+import uk.ac.bbsrc.tgac.miso.core.service.TagBarcodeService;
+
+@Controller
+public class ListTagBarcodesController {
+  @Autowired
+  private TagBarcodeService tagBarcodeService;
+
+  @RequestMapping("/tagbarcodes")
+  public ModelAndView listPools() throws IOException {
```
Comment (callunity) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListTagBarcodesController.java]: `listPools()` -> `listTagBarcodes()` ? :)

Comment (apmasell) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListTagBarcodesController.java]: Oops. Guess where I stole this from.

File: miso-web/src/main/webapp/pages/listTagBarcodes.jsp [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+           <c:forEach items="${tagbarcodes}" var="barcode">[
+            '${barcode.family.platformType.key}', 
+            '${barcode.family.name}',
+            '${barcode.name}',
+            '${barcode.sequence}',
+            '${barcode.family.archived}',
+            '${barcode.id}'
+          ],
+         </c:forEach>
+         ],
```
Comment (callunity) [inline miso-web/src/main/webapp/pages/listTagBarcodes.jsp]: :100:

**Comment by callunity** [comment]
:+1:

**Comment by djcooke** [comment]
:+1:

**Pull request merged**